alfresco-transform-imagemagick/alfresco-transform-imagemagick/src/main/java/org/alfresco/transformer/ImageMagickOptionsBuilder.java

```diff
@@ -107,7 +106,7 @@ public ImageMagickOptionsBuilder withAutoOrient(final String autoOrient)
 
     public ImageMagickOptionsBuilder withAutoOrient(final Boolean autoOrient)
     {
-        this.autoOrient = autoOrient;
+        this.autoOrient = autoOrient == null ? true : autoOrient;
```

Comment:
If null, shouldn't it default to false ?

Reply:
Needs to be true for backward compatibility

Reply:
OK, thanks.